docs/_documentations/troubleshooting.md

```diff
@@ -113,6 +112,23 @@ The Codewind Che extension might lose connectivity to the Codewind pod during a
 
 **Workaround:** Refresh the projects list to have the tree repopulate. If the issue persists, refresh the webpage.
 
+<!--
+Action/Topic: Appsody with Codewind
```

Comment:
Should Appsody here also be removed?

Reply:
Yes, on it...

Reply:
Updated, have replaced that string with the string for the section